deploy/cloudformation/manifest-passthrough.yml

```diff
@@ -0,0 +1,55 @@
+Transform: 'AWS::Serverless-2016-10-31'
+Description: >
+  - Creates a lambda to attach API Keys to Primo API queries.
+Parameters:
+  InfrastructureStackName:
+    Type: String
+    Default: mellon-app-infrastructure
+    Description: The name of the parent infrastructure/networking stack that you created. Necessary
+                 to locate and reference resources created by that stack.
+  AppConfigPath:
+    Type: String
+    Description: The path to look for the application config from
+  PassthroughUrl:
+      Type: String
+      Description: Primo passthrough url
+  PrimoApiKey:
+      Type: String
+      Description: Primo API key
+  PrimoSandboxApiKey:
+      Type: String
+      Description: Sandbox Primo API key
+  Version:
+      Type: String
+      Description: Versioning information
+
+
+Resources:
+  LambdaExecutionRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      RoleName: !Sub "${AWS::StackName}-LambdaExecutionRole"
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: "sts:AssumeRole"
+
+
+  PassthroughLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "${AWS::StackName}-MarblePrimoPassthrough"
+      Handler: index.handler
+      Role: !GetAtt [ LambdaExecutionRole, Arn ]
+      Runtime: "nodejs8.10"
+      Timeout: 90
+      CodeUri: passthrough/
+      Environment:
+        Variables:
+          PRIMO_API_KEY: !Ref PrimoApiKey
+          PASSTHROUGH_URL: !Ref PassthroughUrl
+          PRIMO_SANDBOX_API_KEY: !Ref PrimoSandboxApiKey
```

Comment:
These are clear text in the env. It would be better to have your lambda read these things from AppConfigPath